index.js

```diff
@@ -160,10 +176,18 @@ Use the filterByWord function below to do the following:
     DO NOT USE ADVANCED ARRAY METHODS (i.e. .filter) to solve this problem. 
 */
 
-function filterByWord(/*your code here*/){
-    /*your code here*/
+function filterByWord(originalFlavors, a){
+    const returnOriginalFlavors = [];
+    for(let i = 0; i < originalFlavors.length; i++) {
+        if (originalFlavors[i].includes(a)) {
+            returnOriginalFlavors.push(originalFlavors[i]);
+        }
+    }
+    return returnOriginalFlavors;
 }
 
+console.log(filterByWord(originalFlavors, "chocolate"));
```

Comment:
Overall, fantastic code!  Does this function work if you pass "chocolate" as opposed to "Chocolate"?  You may notice challenges with the first letter being a capital / lower case letter.